ui/raidboss/emulator/ui/Tooltip.ts

```diff
@@ -124,18 +127,6 @@ export default class Tooltip {
     this.tooltip.remove();
   }
 
-  static initializeTemplates(): void {
-    if (Tooltip.templates)
-      return;
-
-    Tooltip.templates = {
-      top: Tooltip.getTemplate('top'),
-      right: Tooltip.getTemplate('right'),
-      bottom: Tooltip.getTemplate('bottom'),
-      left: Tooltip.getTemplate('left'),
-    };
-  }
-
```

Comment:
I think this could originally race condition back when raidemulator was javascript, but that won't happen now that it's all in typescript.